JoonasHeinonen/escape-from-zordoom-prison
# feature/gravity-and-bolt-sparkle: Sparkle added, fall animation and d…

### Link to the ticket: [Gravity and bolt sparkle](https://trello.com/c/WUVQuzGX/19-gravity-and-bolt-sparkle)
![image](https://user-images.githubusercontent.com/33086486/181200906-4c637a64-2480-4e5b-8988-20b31d0d7e91.png)

**Changes:**
 - Added light sparkle to the bolts.
 - Added fall animation for Angela.
 - Gravity reinitialized. Resized the collision box for Angela.
 - Fall death implemented.
 - Added lighting and animation to Angela's accessories, belt and boots.

**How to test:**
 - See that the light sparkle is good enough for our use case.
 - See that the animation for Angela's accessories are working fine.
 - See that the fall animation is working perfectly, currently, it's only 1 frame but should do the trick.
 - See that gravity is working and Angela falls perfectly on the ground. See that she also hits the roof.
 - See that you can no longer do anything after Angela falls from the level. E.g. rotating her to left or right or move her. This means that she has died.
